apps/src/lib/script-editor/LevelToken.jsx

```diff
@@ -99,80 +104,108 @@ class LevelToken extends Component {
+          ({y, scale, shadow}) => (
+            <div
+              style={Object.assign({}, styles.levelToken, {
+                transform: `translate3d(0, ${y}px, 0) scale(${scale})`,
+                boxShadow: `${color.shadow} 0 ${shadow}px ${shadow * 3}px`,
+                zIndex: draggedLevelPos ? 1000 : 500 - this.props.level.position
+              })}
+            >
+              <div style={styles.reorder} onMouseDown={this.handleDragStart}>
+                <i className="fa fa-arrows-v" />
+              </div>
+              <span
+                style={styles.levelTokenName}
+                onMouseDown={this.toggleExpand}
+              >
+                {levelName}
+                {this.props.level.ids.length > 1 && (
+                  <span style={styles.tag}>
+                    {this.props.level.ids.length} variants
+                  </span>
+                )}
+                {this.props.level.challenge && (
+                  <span style={styles.tag}>challenge</span>
+                )}
+                {/* progression supercedes named, so only show the named tag
                   when the level is behaving like a named level. */}
-              {this.props.level.named && !this.props.level.progression && (
-                <span style={styles.tag}>named</span>
-              )}
-              {this.props.level.assessment && (
-                <span style={styles.tag}>assessment</span>
-              )}
-              {this.props.level.progression && (
-                <span style={styles.progressionTag}>
-                  {this.props.level.progression}
-                </span>
+                {this.props.level.named && !this.props.level.progression && (
+                  <span style={styles.tag}>named</span>
+                )}
+                {this.props.level.assessment && (
+                  <span style={styles.tag}>assessment</span>
+                )}
+                {this.props.level.progression && (
+                  <span style={styles.progressionTag}>
+                    {this.props.level.progression}
+                  </span>
+                )}
+              </span>
+              <div style={styles.remove} onMouseDown={this.handleRemove}>
+                <i className="fa fa-times" />
+              </div>
+              {this.props.level.expand && (
+                <LevelTokenDetails
+                  level={this.props.level}
+                  stagePosition={this.props.stagePosition}
+                />
               )}
-            </span>
-            <div style={styles.remove} onMouseDown={this.handleRemove}>
-              <i className="fa fa-times" />
             </div>
-            {this.props.level.expand && (
-              <LevelTokenDetails
-                level={this.props.level}
-                stagePosition={this.props.stagePosition}
-              />
-            )}
-          </div>
+          )}
+        </Motion>
+        {showConfirm && (
```

Comment:
must be outside of `<Motion>` in order for the backdrop to cover the screen. conditionally rendering so as not to introduce 100s of dialog components.